`VirtualSiteHandler` cannot handle `None` keywords

**Describe the bug**

The converters in the `VirtualSiteHandler` introduced in #1252 do not correctly handle the case whereby `outOfPlaneAngle` or `inPlaneAngle` is `"None"` in a serialised force field as this value is passed into the converter as `"None"` rather than `None`.

**To Reproduce**

```
        force_field = ForceField()

        handler = force_field.get_parameter_handler("VirtualSites")
        handler.add_parameter(parameter=VirtualSiteMocking.bond_charge_parameter("[*:1][*:2]"))
        handler.add_parameter(parameter=VirtualSiteMocking.monovalent_parameter("[*:1][*:2][*:3]"))
        handler.add_parameter(parameter=VirtualSiteMocking.divalent_parameter("[*:2][*:1][*:3]", "once"))
        handler.add_parameter(parameter=VirtualSiteMocking.trivalent_parameter("[*:1][*:2][*:3][*:4]"))

        ForceField(force_field.to_string())
```

**Output**

```
        if not supports_out_of_plane_angle and value is not None:
>           raise SMIRNOFFSpecError(
                f"'{self.type}' sites do not support `outOfPlaneAngle`"
E               openff.toolkit.utils.exceptions.SMIRNOFFSpecError: 'BondCharge' sites do not support `outOfPlaneAngle`

```

**Computing environment (please complete the following information):**
 - Operating system
 - Output of running `conda list` 

**Additional context**
<!-- Add any other context about the problem here. -->

--- a/docs/releasehistory.md
+++ b/docs/releasehistory.md
@@ -8,6 +8,12 @@ Releases follow the `major.minor.micro` scheme recommended by [PEP440](https://w
 
 ## Current development
 
+## 0.10.6 Bugfix release
+
+### Critical bugfixes
+
+- [PR #1284](https://github.com/openforcefield/openforcefield/pull/1284): Fixes
+  [Issue #1283](https://github.com/openforcefield/openff-toolkit/issues/1283) - force fields containing `BondCharge` virtual sites cannot be loaded due to an issue with how `outOfPlaneAngle` and `inPlaneAngle` keywords are validated
 
 ## 0.10.5 Bugfix release
 

--- a/openff/toolkit/tests/test_parameters.py
+++ b/openff/toolkit/tests/test_parameters.py
@@ -2088,6 +2088,26 @@ class TestVirtualSiteHandler:
     Test the creation of a VirtualSiteHandler and the implemented VirtualSiteTypes
     """
 
+    def test_serialize_roundtrip(self):
+
+        force_field = ForceField()
+
+        handler = force_field.get_parameter_handler("VirtualSites")
+        handler.add_parameter(
+            parameter=VirtualSiteMocking.bond_charge_parameter("[*:1][*:2]")
+        )
+        handler.add_parameter(
+            parameter=VirtualSiteMocking.monovalent_parameter("[*:1][*:2][*:3]")
+        )
+        handler.add_parameter(
+            parameter=VirtualSiteMocking.divalent_parameter("[*:2][*:1][*:3]", "once")
+        )
+        handler.add_parameter(
+            parameter=VirtualSiteMocking.trivalent_parameter("[*:1][*:2][*:3][*:4]")
+        )
+
+        ForceField(force_field.to_string())
+
     @classmethod
     def build_force_field(cls, v_site_handler: VirtualSiteHandler) -> ForceField:
 

--- a/openff/toolkit/typing/engines/smirnoff/parameters.py
+++ b/openff/toolkit/typing/engines/smirnoff/parameters.py
@@ -4795,6 +4795,9 @@ class VirtualSiteHandler(_NonbondedHandler):
         @outOfPlaneAngle.converter
         def outOfPlaneAngle(self, attr, value):
 
+            if value == "None":
+                return None
+
             supports_out_of_plane_angle = self._supports_out_of_plane_angle(self.type)
 
             if not supports_out_of_plane_angle and value is not None:
@@ -4808,6 +4811,9 @@ class VirtualSiteHandler(_NonbondedHandler):
 
         @inPlaneAngle.converter
         def inPlaneAngle(self, attr, value):
+
+            if value == "None":
+                return None
 
             supports_in_plane_angle = self._supports_in_plane_angle(self.type)
 

